gwcs/converters/selector.py

```diff
@@ -3,35 +3,31 @@
 
 from collections import OrderedDict
 import numpy as np
-from numpy.testing import assert_array_equal
 from astropy.modeling import models
 from astropy.modeling.core import Model
 from astropy.utils.misc import isiterable
 
-from asdf import yamlutil
 from asdf.tags.core.ndarray import NDArrayType
-from ..gwcs_types import GWCSTransformType
+from asdf_astropy.converters.transform.core import TransformConverterBase
 
 
-from ..selector import *
+__all__ = ['LabelMapperConverter', 'RegionsSelectorConverter']
 
 
-__all__ = ['LabelMapperType', 'RegionsSelectorType']
+class LabelMapperConverter(TransformConverterBase):
+    tags = ["tag:stsci.edu:gwcs/label_mapper-*"]
+    types = ["gwcs.selector.LabelMapperArray", "gwcs.selector.LabelMapperDict",
+             "gwcs.selector.LabelMapperRange", "gwcs.selector.LabelMapper"]
 
-
-class LabelMapperType(GWCSTransformType):
-    name = "label_mapper"
-    types = [LabelMapperArray, LabelMapperDict, LabelMapperRange, LabelMapper]
-    version = "1.1.0"
-
-    @classmethod
-    def from_tree_transform(cls, node, ctx):
+    def from_yaml_tree_transform(self, node, tag, ctx):
+        from ..selector import (LabelMapperArray, LabelMapperDict,
+                                LabelMapperRange, LabelMapper)
         inputs_mapping = node.get('inputs_mapping', None)
         if inputs_mapping is not None and not isinstance(inputs_mapping, models.Mapping):
             raise TypeError("inputs_mapping must be an instance"
                             "of astropy.modeling.models.Mapping.")
         mapper = node['mapper']
-        atol = node.get('atol', 10**-8)
+        atol = node.get('atol', 10e-8)
         no_label = node.get('no_label', np.nan)
 
         if isinstance(mapper, NDArrayType):
```

Comment:
The previous error message below about 2x2 masks appears to be inaccurate. Shouldn't it say 2-dimensional masks?

Reply:
Yes, it does appear inaccurate.